CMakeLists.txt

```diff
@@ -42,23 +42,24 @@ option(SWIFT_SYNTAX_ENABLE_WMO_PRE_3_26
        $<IF:$<AND:$<NOT:$<CONFIG:Debug>>,$<PLATFORM_ID:Darwin>>,YES,NO>)
 
 include(AddSwiftHostLibrary)
+include(SwiftCompilerCapability)
```

Comment:
I would suggest to unset `SWIFT_LOADED_MODULE_TRACE_FILE ` like we did in https://github.com/swiftlang/swift/pull/72405 (ideally before we enable the language in line 13)

```suggestion
include(SwiftCompilerCapability)

# we are not interested in logging any Swift module used
# when configuring the build system -- those are not useful
# since they will not contribute to the build of the compiler itself
unset(ENV{SWIFT_LOADED_MODULE_TRACE_FILE})
```

Reply:
Actually, there should not be needed here -- this is important when building `swift-syntax` as part of `swift`, and we are already unsetting that variable there.